js/component-sections.js

```diff
@@ -1,2 +1,39 @@
 // export as global since we're not using a module system
-window.Sections = { template: '<div>Sections {{$route.params.course}} -> {{$route.params.ta}}</div>' }
+window.Sections = Vue.component('Sections',{
+  data(){
```

Comment:
Even though it's valid JS, I would prefer `data: function() {` instead of `data() {`.